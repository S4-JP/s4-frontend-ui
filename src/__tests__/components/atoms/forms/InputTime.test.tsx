import React from 'react'
import { render, screen, fireEvent, waitFor } from '@testing-library/react'
import { InputTime, Color, Time } from '../../../../index'

describe('form - InputTime のレンダリング確認', () => {
  test('オプションなしで設定した場合、初期値でレンダリングされる。', async () => {
    render(<InputTime />)

    expect(screen.getByTestId('atoms-f-input-time-wrap')).toMatchSnapshot()

    expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
      'display: none;'
    )
    // position 設定なしの場合 false
    expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
      'top: 50px;'
    )

    fireEvent.click(screen.getByTestId('atoms-ib-wrap'))
    fireEvent.focus(screen.getByTestId('atoms-f-input-text-styled-input'))

    await waitFor(() =>
      expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
        'display: flex;'
      )
    )

    fireEvent.click(screen.getByTestId('atoms-f-input-time-option-item-0'))
    await waitFor(() =>
      expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
        'display: none;'
      )
    )
  })

  test('props.selectedTimeを設定した場合、該当アイテムの背景色を変更してレンダリングされる。', () => {
    render(<InputTime selectedTime={Time[0]} />)

    expect(screen.getByTestId('atoms-f-input-time-option-item-0')).toHaveStyle(
      `background: ${Color.COMPONENT.WHITE_HOVER}`
    )
    expect(screen.getByTestId('atoms-f-input-time-option-item-1')).toHaveStyle(
      `background: ${Color.COMPONENT.SURFACE}`
    )
  })

  test('props.position に false を設定した場合、optionsにCSSを設定する', () => {
    render(<InputTime position={false} />)

    expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
      'top: 50px;'
    )
  })

  test('props.position に true を設定した場合、optionsにCSSを設定する', () => {
    render(<InputTime position={true} />)

    expect(screen.getByTestId('atoms-f-input-time-options')).toHaveStyle(
      'bottom: 50px;'
    )
  })
})
