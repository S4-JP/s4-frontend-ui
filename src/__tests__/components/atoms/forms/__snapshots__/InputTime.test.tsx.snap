// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form - InputTime のレンダリング確認 オプションなしで設定した場合、初期値でレンダリングされる。 1`] = `
<div
  class="sc-bjUoiL dfGcXs"
  data-testid="atoms-f-input-time-wrap"
>
  <div
    class="sc-jqUVSM iQyIAX"
    data-testid="atoms-f-input-text-wrap"
  >
    <div
      class="sc-kDDrLX cnASSQ"
      data-testid="atoms-f-input-text-input-wrap"
    >
      <input
        class="sc-iqcoie WgzqQ sc-idiyUo izNIWD"
        data-testid="atoms-f-input-text-styled-input"
        value=""
      />
    </div>
  </div>
  <div
    class="sc-fEOsli cEXtHP"
    data-testid="atoms-ib-wrap"
  >
    <span
      class="sc-papXJ ZErXk"
      color="#92A2AF"
      data-testid="atoms-i-icon-wrap"
      size="16"
    >
      <svg
        data-testid="atoms-icon-svg-time"
        viewBox="0 0 40 40"
        xmlns="http://www.w3.org/2000/svg"
      >
        <defs>
          <clippath
            id="time_svg__a"
          >
            <path
              d="M0 0h40v40H0z"
            />
          </clippath>
        </defs>
        <path
          d="M0 0h40v40H0z"
          fill="none"
        />
        <g
          clip-path="url(#time_svg__a)"
        >
          <path
            d="M20 0a20 20 0 1 0 20 20A20 20 0 0 0 20 0m11.877 22.1H20a1.5 1.5 0 0 1-1.5-1.5V7.239a1.5 1.5 0 1 1 3 0V19.1h10.377a1.5 1.5 0 1 1 0 3"
          />
        </g>
      </svg>
    </span>
  </div>
  <div
    class="sc-dIouRR fLEBNN"
    data-testid="atoms-f-input-time-options"
  >
    <label
      for="option_time"
    >
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-0"
      >
        <input
          name="option_time"
          type="radio"
          value="00:00"
        />
        00:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-1"
      >
        <input
          name="option_time"
          type="radio"
          value="00:15"
        />
        00:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-2"
      >
        <input
          name="option_time"
          type="radio"
          value="00:30"
        />
        00:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-3"
      >
        <input
          name="option_time"
          type="radio"
          value="00:45"
        />
        00:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-4"
      >
        <input
          name="option_time"
          type="radio"
          value="01:00"
        />
        01:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-5"
      >
        <input
          name="option_time"
          type="radio"
          value="01:15"
        />
        01:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-6"
      >
        <input
          name="option_time"
          type="radio"
          value="01:30"
        />
        01:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-7"
      >
        <input
          name="option_time"
          type="radio"
          value="01:45"
        />
        01:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-8"
      >
        <input
          name="option_time"
          type="radio"
          value="02:00"
        />
        02:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-9"
      >
        <input
          name="option_time"
          type="radio"
          value="02:15"
        />
        02:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-10"
      >
        <input
          name="option_time"
          type="radio"
          value="02:30"
        />
        02:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-11"
      >
        <input
          name="option_time"
          type="radio"
          value="02:45"
        />
        02:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-12"
      >
        <input
          name="option_time"
          type="radio"
          value="03:00"
        />
        03:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-13"
      >
        <input
          name="option_time"
          type="radio"
          value="03:15"
        />
        03:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-14"
      >
        <input
          name="option_time"
          type="radio"
          value="03:30"
        />
        03:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-15"
      >
        <input
          name="option_time"
          type="radio"
          value="03:45"
        />
        03:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-16"
      >
        <input
          name="option_time"
          type="radio"
          value="04:00"
        />
        04:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-17"
      >
        <input
          name="option_time"
          type="radio"
          value="04:15"
        />
        04:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-18"
      >
        <input
          name="option_time"
          type="radio"
          value="04:30"
        />
        04:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-19"
      >
        <input
          name="option_time"
          type="radio"
          value="04:45"
        />
        04:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-20"
      >
        <input
          name="option_time"
          type="radio"
          value="05:00"
        />
        05:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-21"
      >
        <input
          name="option_time"
          type="radio"
          value="05:15"
        />
        05:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-22"
      >
        <input
          name="option_time"
          type="radio"
          value="05:30"
        />
        05:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-23"
      >
        <input
          name="option_time"
          type="radio"
          value="05:45"
        />
        05:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-24"
      >
        <input
          name="option_time"
          type="radio"
          value="06:00"
        />
        06:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-25"
      >
        <input
          name="option_time"
          type="radio"
          value="06:15"
        />
        06:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-26"
      >
        <input
          name="option_time"
          type="radio"
          value="06:30"
        />
        06:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-27"
      >
        <input
          name="option_time"
          type="radio"
          value="06:45"
        />
        06:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-28"
      >
        <input
          name="option_time"
          type="radio"
          value="07:00"
        />
        07:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-29"
      >
        <input
          name="option_time"
          type="radio"
          value="07:15"
        />
        07:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-30"
      >
        <input
          name="option_time"
          type="radio"
          value="07:30"
        />
        07:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-31"
      >
        <input
          name="option_time"
          type="radio"
          value="07:45"
        />
        07:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-32"
      >
        <input
          name="option_time"
          type="radio"
          value="08:00"
        />
        08:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-33"
      >
        <input
          name="option_time"
          type="radio"
          value="08:15"
        />
        08:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-34"
      >
        <input
          name="option_time"
          type="radio"
          value="08:30"
        />
        08:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-35"
      >
        <input
          name="option_time"
          type="radio"
          value="08:45"
        />
        08:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-36"
      >
        <input
          name="option_time"
          type="radio"
          value="09:00"
        />
        09:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-37"
      >
        <input
          name="option_time"
          type="radio"
          value="09:15"
        />
        09:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-38"
      >
        <input
          name="option_time"
          type="radio"
          value="09:30"
        />
        09:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-39"
      >
        <input
          name="option_time"
          type="radio"
          value="09:45"
        />
        09:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-40"
      >
        <input
          name="option_time"
          type="radio"
          value="10:00"
        />
        10:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-41"
      >
        <input
          name="option_time"
          type="radio"
          value="10:15"
        />
        10:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-42"
      >
        <input
          name="option_time"
          type="radio"
          value="10:30"
        />
        10:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-43"
      >
        <input
          name="option_time"
          type="radio"
          value="10:45"
        />
        10:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-44"
      >
        <input
          name="option_time"
          type="radio"
          value="11:00"
        />
        11:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-45"
      >
        <input
          name="option_time"
          type="radio"
          value="11:15"
        />
        11:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-46"
      >
        <input
          name="option_time"
          type="radio"
          value="11:30"
        />
        11:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-47"
      >
        <input
          name="option_time"
          type="radio"
          value="11:45"
        />
        11:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-48"
      >
        <input
          name="option_time"
          type="radio"
          value="12:00"
        />
        12:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-49"
      >
        <input
          name="option_time"
          type="radio"
          value="12:15"
        />
        12:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-50"
      >
        <input
          name="option_time"
          type="radio"
          value="12:30"
        />
        12:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-51"
      >
        <input
          name="option_time"
          type="radio"
          value="12:45"
        />
        12:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-52"
      >
        <input
          name="option_time"
          type="radio"
          value="13:00"
        />
        13:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-53"
      >
        <input
          name="option_time"
          type="radio"
          value="13:15"
        />
        13:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-54"
      >
        <input
          name="option_time"
          type="radio"
          value="13:30"
        />
        13:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-55"
      >
        <input
          name="option_time"
          type="radio"
          value="13:45"
        />
        13:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-56"
      >
        <input
          name="option_time"
          type="radio"
          value="14:00"
        />
        14:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-57"
      >
        <input
          name="option_time"
          type="radio"
          value="14:15"
        />
        14:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-58"
      >
        <input
          name="option_time"
          type="radio"
          value="14:30"
        />
        14:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-59"
      >
        <input
          name="option_time"
          type="radio"
          value="14:45"
        />
        14:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-60"
      >
        <input
          name="option_time"
          type="radio"
          value="15:00"
        />
        15:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-61"
      >
        <input
          name="option_time"
          type="radio"
          value="15:15"
        />
        15:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-62"
      >
        <input
          name="option_time"
          type="radio"
          value="15:30"
        />
        15:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-63"
      >
        <input
          name="option_time"
          type="radio"
          value="15:45"
        />
        15:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-64"
      >
        <input
          name="option_time"
          type="radio"
          value="16:00"
        />
        16:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-65"
      >
        <input
          name="option_time"
          type="radio"
          value="16:15"
        />
        16:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-66"
      >
        <input
          name="option_time"
          type="radio"
          value="16:30"
        />
        16:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-67"
      >
        <input
          name="option_time"
          type="radio"
          value="16:45"
        />
        16:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-68"
      >
        <input
          name="option_time"
          type="radio"
          value="17:00"
        />
        17:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-69"
      >
        <input
          name="option_time"
          type="radio"
          value="17:15"
        />
        17:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-70"
      >
        <input
          name="option_time"
          type="radio"
          value="17:30"
        />
        17:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-71"
      >
        <input
          name="option_time"
          type="radio"
          value="17:45"
        />
        17:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-72"
      >
        <input
          name="option_time"
          type="radio"
          value="18:00"
        />
        18:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-73"
      >
        <input
          name="option_time"
          type="radio"
          value="18:15"
        />
        18:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-74"
      >
        <input
          name="option_time"
          type="radio"
          value="18:30"
        />
        18:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-75"
      >
        <input
          name="option_time"
          type="radio"
          value="18:45"
        />
        18:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-76"
      >
        <input
          name="option_time"
          type="radio"
          value="19:00"
        />
        19:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-77"
      >
        <input
          name="option_time"
          type="radio"
          value="19:15"
        />
        19:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-78"
      >
        <input
          name="option_time"
          type="radio"
          value="19:30"
        />
        19:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-79"
      >
        <input
          name="option_time"
          type="radio"
          value="19:45"
        />
        19:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-80"
      >
        <input
          name="option_time"
          type="radio"
          value="20:00"
        />
        20:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-81"
      >
        <input
          name="option_time"
          type="radio"
          value="20:15"
        />
        20:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-82"
      >
        <input
          name="option_time"
          type="radio"
          value="20:30"
        />
        20:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-83"
      >
        <input
          name="option_time"
          type="radio"
          value="20:45"
        />
        20:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-84"
      >
        <input
          name="option_time"
          type="radio"
          value="21:00"
        />
        21:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-85"
      >
        <input
          name="option_time"
          type="radio"
          value="21:15"
        />
        21:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-86"
      >
        <input
          name="option_time"
          type="radio"
          value="21:30"
        />
        21:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-87"
      >
        <input
          name="option_time"
          type="radio"
          value="21:45"
        />
        21:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-88"
      >
        <input
          name="option_time"
          type="radio"
          value="22:00"
        />
        22:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-89"
      >
        <input
          name="option_time"
          type="radio"
          value="22:15"
        />
        22:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-90"
      >
        <input
          name="option_time"
          type="radio"
          value="22:30"
        />
        22:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-91"
      >
        <input
          name="option_time"
          type="radio"
          value="22:45"
        />
        22:45
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-92"
      >
        <input
          name="option_time"
          type="radio"
          value="23:00"
        />
        23:00
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-93"
      >
        <input
          name="option_time"
          type="radio"
          value="23:15"
        />
        23:15
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-94"
      >
        <input
          name="option_time"
          type="radio"
          value="23:30"
        />
        23:30
      </label>
      <label
        class="sc-hHLeRK cteoPY"
        data-testid="atoms-f-input-time-option-item-95"
      >
        <input
          name="option_time"
          type="radio"
          value="23:45"
        />
        23:45
      </label>
    </label>
  </div>
</div>
`;
